openedx/features/edly_panel/tests.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from django.test import TestCase
+from .models import EdlyActiveUser
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class EdlyActiveUserModelTest(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create(username='test@example.com', password='test_password')
+        EdlyActiveUser.objects.create(user=self.user)
+
+    def test_string_representaion(self):
+        active_user = EdlyActiveUser.objects.filter(user=self.user, created_at__date=timezone.now().date())
+        self.assertIn('{} was active on {}'.format(self.user.username, timezone.now().date()), str(active_user))
```

Comment:
We can improve the readability like this:
```python
active_user = EdlyActiveUser.objects.filter(user=self.user).first()
expected_string = '{username} was active on {active_date}'.format(
    username=self.user.username, active_date=timezone.now().date()
)
self.assertEqual(expected_string, str(active_user))
```

Reply:
done.